app/views/layouts/home.html.erb

```diff
@@ -29,7 +29,7 @@
       <div id="pageBody" class="container">
 
         <!-- Flashes -->
-        <div id='flashes'>
+        <div id="flashes">
           <% flash.each do |name, msg| %>
           <%= content_tag :div, raw(msg), id: "flash_#{name}",
```

Comment:
Sanitize here, while we're at it. (Actually, you could also make a partial template for this, but you don't need to)